src/_includes/partials/helpus.njk

```diff
@@ -1,33 +1,35 @@
-<hr />
+<section class="py-400 mt-500 bt-sm text-secondary">
+	
+	<h2 class="text-primary">
+	{{ helpus[locale].heading }}
+	</h2>
 
-<h2 class="mt-500 mb-400">
-  {{ helpus[locale].heading }}
-</h2>
+	<p class="mt-400 text-primary">
+		{{ helpus[locale].paragraph }}
+	</p>
 
-<p class="mb-400">
-	{{ helpus[locale].paragraph }}
-</p>
+	<ul class="mt-400 md:d-flex gap-250 text-primary">
+		<li class="py-400">
```

Comment:
I think button of type link should display as block. This `py-400` is kind of hacky at the moment because the buttons are inline. That change needs to happen in gcds-components though.

Reply:
Most links are normally used inline so it makes sense to keep it as `inline-block`.
When you change the button to `display: block;` in inspect, it doesn't have any effect on the layout.

This change would make more sense for button type buttons. We could add a prop to toggle between `display: block` and `display: inline-block;` to the component in the future, but it's not a high priority for now. I will make a note of this for future updates after the release!

Reply:
inline-block makes more sense yes. But This is not something I can do here anyways

Reply:
I disagree with the `display: block` point. Even though the button is a link, we want it to display similar to how how buttons do, which by default is inline-block. 

Also did this list need to be updated? Was rendering fine before any changes.

Reply:
I removed all my changes. I just noticed some spacing issues in the documentation pages, but could be fixed later